spec/features/lock_user_account_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'User account gets locked', type: :feature do
```

Comment:
Woohoo!  Thank you for this awesome feature spec coverage! 💃 🙏 👏 😁

Reply:
Ah! I was losing my mind trying to figure out why the messages weren't showing up which lead to the sessions controller solution. Thanks so much for picking that up. 👍  I'll push up the changes soon!